ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -315,10 +315,16 @@ func (s *indexerStreamingService) ListenCommit(ctx context.Context, res abci.Res
 	defer func() {
```

Comment:
In v25.x, it's observed that `OnWrite` is triggered after `ListenEndBlock` and so we don't reset the `poolTracker` until we obtain BOTH newly created pool ID, as well as its pool metadata.
In v26, I wonder if we have to do something similar and thus the reset timing here might need adjustment. What do you think? Certainly, we will know more when we put it into the testnet environment. Maybe you have some insights already?

Reply:
In v26.x, we apply the kv store listened changes (change set) in ListenCommit() so we should avoid resetting the pool tracker until end of ListenCommit.

The current behavior on this branch looks consistent with the above where we reset it in the defer statement of ListenCommit

Reply:
Note that ListenCommit is called after ListenEndBlock and before the next block begins